### ISSUE TITLE
NVDA repeats name of component after moving focus out of application role

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/a11y-research#201]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Firefox + NDVA

**Problem description**
From https://github.com/phetsims/qa/issues/1314
Screen reader says twice button unavailable: "_Button unavailable GO,  GO, Button unavailable_ "

**Visuals**

https://github.com/user-attachments/assets/0a2d7c9e-7c2c-495d-9ee4-ac06d4b4d53c

**Comment by samreid** [comment]
I cannot reproduce the problem on macOS + voiceover on chrome. @jessegreenberg can you please advise how to proceed?

**Comment by vperezcadavid99** [comment]
I don't know why, but it doesn't always happen. I just tried it on Chrome, and I found a way to reproduce it:

1. Navigate with the tab to the "Net force" screen.
2. Press tab again to highlight the blue pullers panel. "_Large blue puller sortable."_
3. Press tab again to highlight the red pullers panel. "_small red puller 1 sortable._"
4. Press tab again to highlight the Go Button. "_Heading level 1 Net Force screen..._"
5. Press shift+tab back to "_small red puller 1 sortable._"
6. Press tab again to highlight the Go Button. "_Button unavailable Go, Go, Button unavailable._"


https://github.com/user-attachments/assets/77e079bc-4f4e-4eb9-b9e1-5a82a9b38967

**Comment by jessegreenberg** [comment]
Thanks @vperezcadavid99, I was able to reproduce this quickly while tabbing to the go button in Chrome. This is pretty basic navigation so Ill look for anything strange in the markup and then try to get it to happen outside of the sim.

**Comment by jessegreenberg** [comment]
Here is the element with various scenery data removed:

```
<button aria-disabled="true" onclick="return false">Go!</button>
```

**Comment by jessegreenberg** [comment]
I was able to reproduce the problem with the following markup:

```html
<head>
  <title>Testing</title>
</head>
<body>
<div role="application">
  <button>B1</button>
  <button>B2</button>
  <button>B3</button>
</div>
<button>Go!</button>
</body>
```

The problem only started to happen when I added teh `<div role=application>`. So this is an NVDA bug related to moving focus out of the application role. Ill move this to a11y-research and submit a bug report to NVDA. It no longer blocks forces-and-motion-basics.

Thanks @vperezcadavid99 !
